google/cloud/aiplatform/datasets/dataset.py

```diff
@@ -197,10 +212,11 @@ def create(
             display_name=display_name,
             metadata_schema_uri=metadata_schema_uri,
             datasource=datasource,
-            project=project or initializer.global_config.project,
```

Comment:
Removed as this is redundant. The global_config's are used downstream if not already set.

Reply:
Double check this because this is a synchronous method calling into an optionally asynchronous method meaning the value of global_config could have changed between the call time of `create` and the execution time of `_create_and_import`.